Nevolution/decorator-wechat
# Update emoji to latest version

Update the emoji to support latest WeChat 8
- Added missing WeChat emoji that came with new versions
- Add in dup/changed codes
- Change some emoji to be more consistent with meaning and/or WeChat style
- Add and update PC emoji as well
- Also taking advantage of new emoji that was released since the last time this was updated

How to verify changed emoji:
-> I went through every 安卓 wechat emoji and 电脑微信 emoji and verified their codes one-by-one
\- Use WeChat 8.0 and verify all codes from changed emoji
\- Use PC WeChat and verify all codes from changed emoji
-> Took a long process of having to send each emoji to myself to verify both the Chinese code and English code
